src/themes/ec/variables/_language-list.scss

```diff
@@ -15,8 +15,17 @@ $language-list: (
     inline-start: var(--s-xl),
     inline-end: var(--s-xl),
   ),
-  header-font-mobile: var(--f-2xl),
-  header-font-desktop: var(--f-3xl),
+  header-typography: (
```

Comment:
font seems different (mobile and desktop)

Reply:
changed, thanks

Reply:
I would have changed more, to simple heading approach as param, but the heading in ec have font-weight light as default.. and would not be the same.